.mailmap

```diff
@@ -245,6 +249,7 @@ Shanggao Qiu <qiushanggao@qq.com> qiushanggao
 Shawn Chen <cxwshawn@gmail.com> <cxwshawn@gmail.com>
 Shu, Xinxin <xinxin.shu@intel.com> xinxin shu <xinxin.shu@intel.com>
 Shu, Xinxin <xinxin.shu@intel.com> xinxinsh <xinxin.shu@intel.com>
+Song Baisen <song.baisen@zte.com.cn>
```

Comment:
Le 04/02/16 14:58, Abhishek L a écrit :
>
> In .mailmap <https://github.com/ceph/ceph/pull/7517#discussion_r51874918>:
>
> > @@ -245,6 +249,7 @@ Shanggao Qiu <qiushanggao@qq.com> qiushanggao
> >  Shawn Chen <cxwshawn@gmail.com> <cxwshawn@gmail.com>
> >  Shu, Xinxin <xinxin.shu@intel.com> xinxin shu <xinxin.shu@intel.com>
> >  Shu, Xinxin <xinxin.shu@intel.com> xinxinsh <xinxin.shu@intel.com>
> > +Song Baisen <song.baisen@zte.com.cn>
>
> Does an entry like this change the output or so.. As far the author 
> has used only one email, I guess this entry could be dropped
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/ceph/ceph/pull/7517/files#r51874918>.
>
BTW, I was reading https://git-scm.com/docs/git-shortlog , .mailmap 
section, and I have the feeling our .mailmap is over complicated.

For example, in my yesterday pull request,  patch 
64c0e25673d1e11301713b0f51ad136ae9802990 I had to made a fix for sage 
typo in his name (sage Weli) but email is good.

This one is not needed, a single Sage Weil <sweil@redhat.com> should be 
sufficient, all variant could be eliminated.
I've made some of these errors in the past, and I'm quite sure we can 
somewhat simplify this file.

I'll try this if you're ok with that.